Classes/GraphQL/Resolver.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flowpack\Media\Ui\GraphQL;
+
+/*
+ * This file is part of the Flowpack.Media.Ui package.
+ *
+ * It is based on the wwwision/neos-graphql package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use BackedEnum;
+use GraphQL\Language\AST\EnumTypeDefinitionNode;
+use GraphQL\Language\AST\InputObjectTypeDefinitionNode;
+use GraphQL\Language\AST\InterfaceTypeDefinitionNode;
+use GraphQL\Language\AST\TypeDefinitionNode;
+use GraphQL\Server\RequestError;
+use GraphQL\Type\Definition\Argument;
+use GraphQL\Type\Definition\FieldDefinition;
+use GraphQL\Type\Definition\ListOfType;
+use GraphQL\Type\Definition\NonNull;
+use GraphQL\Type\Definition\ResolveInfo;
+use InvalidArgumentException;
+use UnitEnum;
+use Wwwision\Types\Exception\CoerceException;
+use Wwwision\Types\Parser;
+use Wwwision\Types\Schema\EnumCaseSchema;
+use Wwwision\Types\Schema\EnumSchema;
+use Wwwision\Types\Schema\ShapeSchema;
+use Wwwision\TypesGraphQL\Types\CustomResolvers;
+
+use function Wwwision\Types\instantiate;
+
+class Resolver
```

Comment:
puuh im not really sure what this does exactly but id love a little testing or docs here for such framework code 🙈